### DISCUSSION TITLE
alevin: the --whitelist flag

### DISCUSSION THREAD

**Discussion by eudoraleer** [COMBINE-lab/salmon#646] [Q&A (single-cell specific)]
Hi,

I saw this flag on salmon alevin:
<img width="730" alt="image" src="https://user-images.githubusercontent.com/5945741/114036302-3e191880-9880-11eb-9946-742183f86134.png">

I wonder does it mean we cannot use 10x whitelist as input to this flag?

Thanks!

**Comment by k3yavi** [comment]
Hi @eudoraleer ,

Thanks for checking. That's correct, this flag is for providing a small list of cellular barcodes which you'd like to quantify. A general use case is re-quantification of the Cellranger (or any tool) quantified data and user knows what high quality of barcodes (typically in ~10k) needs to quantified. Alevin will take the list and quantify only the provided barcodes, post cellular barcode correction.

**Reply by eudoraleer** [reply]
Hi!

Thank you for your quick reply. Does Alevin does barcode correction without user specifying input to this --whitelist flag?

Also, just a quick question, how about 10XV3 whitelist as input to --whitelist flag? Or this whitelist flag is not used for this purpose? Thanks for explaining, I am a bit confused.
